### PR TITLE
LRAC-8769, LRAC-8944 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRAC-8769
https://issues.liferay.com/browse/LRAC-8944

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DocumentsAndMediaEvents.testcase [hunk 1]
```diff
@@ -0,0 +1,249 @@
+@component-name = "portal-analytics-cloud"
+definition {
+
+	property analytics.cloud.enabled = "true";
+	property custom.properties = "jsonws.web.service.paths.excludes=";
+	property minimum.slave.ram = "32";
+	property portal.release = "false";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Analytics Cloud";
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DocumentsAndMediaEvents.testcase:9]: Hi @melodywu, I think you have some wrong properties in your file. Could you put it like this?

```
property analytics.cloud.enabled = "true";
property minimum.slave.ram = "24";
property portal.release = "false";
property portal.upstream = "true";
property slave.label = "slave-1";
property testray.main.component.name = "Analytics Cloud";
```

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
hi @melodywu , this PR is failing on sf because you have a commit missing the LRAC-id. Could you please review it? Thanks!

**Comment by melodywu** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
hi @ViniLopes20  , thanks for helping to review it.

Hi @melodywu , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**